mpp/mpp_domains.F90

```diff
@@ -442,10 +450,20 @@ module mpp_domains_mod
 
   end type nestSpec
 
-
-
   type nest_domain_type
+     character(len=NAME_LENGTH)     :: name
```

Comment:
Where is the documentation for the nest_domain_type and nest_level_type?  Documentation of the derived types is something we should _always_ have.